tests/sles4sap/redirection_tests/ensa2_hanadb_sapcontrol_move_ascs.pm

```diff
@@ -0,0 +1,117 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Test module tests ENSA2 Central Services with HANA DB - Use sapcontrol to move ASCS.
+#   It runs sapcontrol related commands on remote host using console redirection.
+#   For more information read 'README.md'
+
+use parent 'sles4sap::sap_deployment_automation_framework::basetest';
+
+use warnings;
+use strict;
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use sles4sap::console_redirection;
+use hacluster qw(check_cluster_state wait_until_resources_started wait_for_idle_cluster crm_check_resource_location);
+use sles4sap::sap_deployment_automation_framework::deployment qw(load_os_env_variables get_sdaf_instance_id);
+use sles4sap::sapcontrol qw(sapcontrol sap_show_status_info);
+use saputils;
+use Data::Dumper;
+
+sub webmethod_checks {
+    my ($instance_id, $sidadm) = @_;
+    my $outputs;
+    my $looptime = 300;
+
+    # General status will help with troubleshooting
+    sap_show_status_info(cluster => 1, netweaver => 1, instance_id => $instance_id);
+    record_info('ENSA check', "Executing 'HACheckConfig' and 'HACheckFailoverConfig'");
+    while ($outputs = sapcontrol(webmethod => 'HACheckConfig', instance_id => $instance_id, sidadm => $sidadm, return_output => 1)) {
+        last unless ($outputs =~ /ERROR|FAIL/);
+        record_info("ERROR found in HACheckConfig: $outputs", "sleep 30s and try again");
+        sleep 30;
+        $looptime -= 30;
+        last if ($looptime <= 0);
+    }
+    sapcontrol(webmethod => 'HACheckFailoverConfig', instance_id => $instance_id, sidadm => $sidadm);
+}
```

Comment:
Doesn't it make more sense to have this in a library? I suppose you'll need it to also check ERS failover later.

Reply:
> Doesn't it make more sense to have this in a library? I suppose you'll need it to also check ERS failover later.

Moved to lib file and new VRs are passed.